specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2048,14 +2059,15 @@
                 "ports"
             ]
         },
-        "JitNetworkAccessPolicyActionPort": {
+        "JitNetworkAccessPolicyInitiatePort": {
             "type": "object",
             "properties": {
                 "number": {
                     "$ref": "#/definitions/PortNumber"
                 },
                 "allowedSourceAddressPrefix": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Source of the allowed traffic. if omitted, the request will be for the source IP address of the initiate reuqest"

```

Comment:
Some grammar and typo correction:
`"Source of the allowed traffic. If omitted, the request will be for the source IP address of the initiate request." `

Reply:
Resolved